yt/yt/tests/integration/misc/test_logging.py

```diff
@@ -0,0 +1,166 @@
+from yt_env_setup import YTEnvSetup
+
+from yt_commands import authors, wait, create, sync_mount_table, sync_create_cells, select_rows, start_transaction, set
+
+import time
+
+##################################################################
+
+
+class TestDynamicTableLogger(YTEnvSetup):
+    NUM_MASTERS = 3
+    NUM_NODES = 1
+    USE_DYNAMIC_TABLES = True
+
+    DEBUG_LOG_TABLE = "//tmp/debug_log"
+    ACCESS_LOG_TABLE = "//tmp/access_log"
+
+    DELTA_MASTER_CONFIG = {
+        "logging": {
+            "flush_period": 100,
+            "shutdown_busy_timeout": 1000,
+            "rules": [
+                {
+                    "family": "plain_text",
+                    "min_level": "debug",
+                    "exclude_categories": ["Bus", "Concurrency"],
+                    "writers": ["debug_dynamic_table"],
+                },
+                {
+                    "family": "structured",
+                    "min_level": "debug",
+                    "include_categories": ["Access"],
+                    "writers": ["access_dynamic_table"]
+                },
+            ],
+            "writers": {
+                "debug_dynamic_table": {
+                    "type": "dynamic_table",
+                    "format": "yson",
+                    "enable_host_field": True,
+                    # We need to explicitly specify this for plain text yson logs.
+                    "enable_system_messages": True,
+                    # We need to specify this to format system messages correctly.
+                    "system_message_family": "plain_text",
+                    "table_path": DEBUG_LOG_TABLE,
+                    "flush_period": 100,
+                    # Helps skip messages logged while dynamic table is not created yet.
+                    "write_backoff": {
+                        "invocation_count": 1,
+                    },
+                },
+                "access_dynamic_table": {
+                    "type": "dynamic_table",
+                    "format": "yson",
+                    "enable_host_field": True,
+                    "table_path": ACCESS_LOG_TABLE,
+                    "common_fields": {
+                        "cluster": "primary",
+                    },
+                    "flush_period": 400,
+                },
+            },
+            # This is specified to test rate limits.
+            "category_rate_limits": {
+                # No sequoia logs for you!
+                "SequoiaServer": 0,
```

Comment:
It may be wise to use another category, since `Sequoia` logs are relatively new and are not present in 23.2, for example.

Reply:
Changed to Concurrency.